Userland/Utilities/dmesg.cpp

```diff
@@ -11,10 +11,10 @@
 ErrorOr<int> serenity_main(Main::Arguments)
 {
     TRY(Core::System::pledge("stdio rpath"));
-    TRY(Core::System::unveil("/proc/dmesg", "r"));
+    TRY(Core::System::unveil("/sys/kernel/log", "r"));
```

Comment:
Any particular reason to rename to `/sys/kernel/log` rather than `/sys/kernel/dmesg` ? The name change of the file makes the name of the utility less convincing.